### PR TITLE
feat(invoke): Update invoke to v2.x version

### PR DESCRIPTION
**Pull request by kissifrot**
Fixes compatibility with Python 3.11

### CODE REVIEW

File: invoke.py [hunk 1]
```diff
@@ -36,7 +36,7 @@ def __extract_runtime_configuration(config):
     from colorama import init, Fore
     init(autoreset=True)
 
-    config['root_dir'] = os.path.dirname(os.path.abspath(__file__))
+    config['root_dir'] = os.path.dirname(os.path.abspath("__file__"))
```
Comment (lyrixx) [inline invoke.py:39]: This looks weird. Are you sure?

Comment (kissifrot) [reply invoke.py]: Sorry I failed my stash, I pushed an earlier non-working version

Comment (lyrixx) [reply invoke.py]: Is the PR okay now?

Comment (kissifrot) [reply invoke.py]: Yes it is, sorry again :sweat_smile:

**Review by lyrixx — APPROVED** [review]
Thanks 👍🏼

**Pull request merged**